Eatery Blue/UI/Profile/ProfileLoginViewController.swift

```diff
@@ -79,21 +78,9 @@ class ProfileLoginViewController: UIViewController {
         view.addSubview(eateryLogo)
     }
 
-    private func setUpScrollView() {
-        scrollView.showsHorizontalScrollIndicator = false
-        scrollView.showsVerticalScrollIndicator = false
-        scrollView.isPagingEnabled = true
-
-        scrollView.tap { [self] _ in
-            view.endEditing(true)
-        }
-
-        scrollView.addSubview(loginView)
-        setUpLoginView()
-    }
-
     private func setUpLoginView() {
         loginView.setCustomSpacing(8)
+        loginView.addTitleLabel("Log in with Eatery")
```

Comment:
Wait is this following Kathleen's design? Kinda weird that it's Eatery and not like GET or Duo

Reply:
Yea. I think Eatery makes more sense tho even if we use Cornell SSO login just bc it's our app.